docs/config/global/README.md

```diff
@@ -14,19 +14,28 @@ Help Options:
-  --enable-unassign-security-groups=[true|false]  Enable unassigning security groups
-  --metadata-prefix=                              Prefix for org/space metadata
-  --staging-security-group=                       Staging Security Group to add
-  --remove-staging-security-group=                Staging Security Group to remove
-  --running-security-group=                       Running Security Group to add
-  --remove-running-security-group=                Running Security Group to remove
-  --shared-domain=                                Shared Domain to add
-  --router-group-shared-domain=                   Router Group Shared Domain to add
-  --router-group-shared-domain-group=             Router Group Shared Domain group
-  --internal-shared-domain=                       Internal Shared Domain to add
-  --remove-shared-domain=                         Shared Domain to remove
+          --config-dir=                                   Name of the config directory (default: config) [$CONFIG_DIR]
+          --enable-delete-isolation-segments=[true|false] Enable removing isolation segments
+          --enable-delete-shared-domains=[true|false]     Enable removing shared domains
+          --enable-service-access=[true|false]            Enable managing service access
+          --enable-unassign-security-groups=[true|false]  Enable unassigning security groups
+          --metadata-prefix=                              Prefix for org/space metadata
+          --staging-security-group=                       Staging Security Group to add
+          --remove-staging-security-group=                Staging Security Group to remove
+          --running-security-group=                       Running Security Group to add
+          --remove-running-security-group=                Running Security Group to remove
+          --shared-domain=                                Shared Domain to add
+          --router-group-shared-domain=                   Router Group Shared Domain to add
+          --router-group-shared-domain-group=             Router Group Shared Domain group
+          --internal-shared-domain=                       Internal Shared Domain to add
+          --remove-shared-domain=                         Shared Domain to remove
+
+    service-access:
+          --broker=                                       Name of Broker
+          --service=                                      Name of Service
+          --all-access-plan=                              Plan to give access to all orgs
+          --limited-access-plan=                          Plan to give limited access to, must also provide org list
+          --org=                                          Orgs to add to limited plan
+          --remove-org=                                   Orgs to remove from limited plan
+          --no-access-plan=                               Plan to give access to all orgs
```

Comment:
The description doesn't seem to apply to a no-access-plan flag. Description looks like a duplicate of --all-access-plan

Reply:
Good catch. I'll remove this commit from the PR and open another one. We reused the output from `cf-mgmt-config global --help` assuming it was correct. It looks like it needs some attention.